infrastructure/terraform/README.md

```diff
@@ -17,6 +17,12 @@ git clone  https://github.com/Azure/ai-hub.git
 
 ## Local Setup
 
+Make sure you replace:
+
+-  _azureregion_ with your desired Azure region (for example, swedencentral)
+- _xx-XX_ with the default language of the content to be processed (for example, en-US or es-ES)
+- _yourprefix_ with your desired 8-character prefix. 
```

Comment:
Other MSFT doc usually use something like `"<your-azure-region>"` or `"{your-azure-region}"`. Should we do the same?